docs/faq/end-users.md

```diff
@@ -1,49 +1,84 @@
-# End users
+# End User FAQ
 
-> Hermez is currently under development. Some of the details in the answers can be modified before network launch.
+# Overview & Getting Started 
 
-## How much is a transaction expected to cost?
+# What exactly is Hermez?
 
-While we don't yet have an exact figure, our best estimate is that a normal transaction should cost on the order few dollar cents. The price will be determined by the coordinators but this price will depend amongst other on the L1 gas price and the operation costs of coordinators but we estimate a reduction around 95% of the current L1 transaction cost.
+Hermez is a Layer 2 solution to scale payments on top of Ethereum.  It works by grouping transactions together to create SNARKs - succinct non-interactive arguments of knowledge.  These SNARKs then get settled on the Ethereum base layer, Layer 1, as one transaction. The transactions are executed by Coordinators (our version of block producers).  This means they are effectively running the network by computing the zero-knowledge proof of validity of the transactions made by users.  The results is low-cost token transfers, with all the security of Ethereum.  
 
-## Which tokens can I transact with?
+# How do I set up my Hermez Wallet? 
 
-While Hermez can support any ERC-20 token, it's primarily designed with high-frequency tokens like ETH, DAI, Tether, and wBTC in mind.
+From a desktop computer, please visit the <a href="https://hermez.io/wallet/"> Hermez Wallet </a> landing page and follow the instructions for setup. 
 
-## How long does a transaction take?
 
-While a coordinator will receive a transaction almost instantly, they need to forge the blocks and generate the zk-proofs as a first stage.
+# Transactions
 
-Since both data and zk-proofs of correctness need to be included on the Ethereum base-chain, the time for data to be reliably included (ideally finalized) on-chain depends on the Ethereum block time.
+# How long do transfers take on Hermez Network from start to finish? 
```

Comment:
Change to ### How long do transfers take on Hermez Network from start to finish?